src/main/java/net/p455w0rd/wirelesscraftingterminal/common/container/ContainerWirelessCraftingTerminal.java

```diff
@@ -1317,6 +1317,8 @@ public ItemStack slotClick(int slot, int button, int flag, EntityPlayer player)
                     && getSlot(slot) != null
                     && getSlot(slot).getStack() == RandomUtils.getWirelessTerm(player.inventory)) {
                 return null;
+            }else if(player.inventory.currentItem == button && flag == 2){
```

Comment:
you need to format your code like this to pass the spotless check 
![image](https://user-images.githubusercontent.com/57050655/190576926-a3ed2c4e-6976-47ed-a310-c908b00603c4.png)

or in your IDE you can run the spotlessJavaApply task
![image](https://user-images.githubusercontent.com/57050655/190576983-cb41cb89-deeb-4173-b110-56eafbe26dc3.png)